src/main/java/net/logstash/logback/stacktrace/ShortenedThrowableConverter.java

```diff
@@ -150,7 +152,14 @@
      * True to print the root cause first.  False to print exceptions normally (root cause last).
      */
     private boolean rootCauseFirst;
-    
+
+    /**
+     * True to compute and inline stack hashes.
+     */
+    private boolean inlineHash;
```

Comment:
You're right.

I saw the `parseOptions` thing, but don't really understand how it works (didn't find any doc about it)

But I'll give it a look